tests/test_nodes.py

```diff
@@ -64,7 +65,8 @@ def test_selection_working(snapshot_custom: NumpySnapshotExtension, attribute, c
 @pytest.mark.parametrize("code", codes)
 @pytest.mark.parametrize("attribute", ["chain_id", "entity_id"])
 def test_color_custom(snapshot_custom: NumpySnapshotExtension, code, attribute):
-    mol = mn.entities.fetch(code, style="ribbon", cache_dir=data_dir)
+    mol = mn.Molecule.fetch(code, cache=data_dir)
+    mol.add_style("ribbon")
```

Comment:
I prefer `Set style` to `Add_style` .  `Add_style` implies more than one style.

Reply:
I'm going with `add_style()` so that we can add multiple styles on the same Molecule, which will just be additional branches in the node tree. Currently calling `add_style()` once will be equivalent to the previous style setyp.